presto-native-execution/presto_cpp/main/tests/PrestoTestServer.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+#include "presto_cpp/main/tests/PrestoTestServer.h"
+#include "velox/functions/prestosql/aggregates/RegisterAggregateFunctions.h"
+#include "velox/functions/prestosql/registration/RegistrationFunctions.h"
+
+namespace facebook::presto {
+using namespace facebook::velox;
+
+PrestoTestServer::PrestoTestServer(const std::string& configDirectoryPath)
```

Comment:
Looking at this codebase, I see several examples where cpp files have multiple classes so I assume that having a dedicated file for cpp isn't a best practice, rather a dev discretion.

If that assumption holds, would it make sense to fold this class into PrestoTestMain? To me, it makes sense given how small the class is.

Reply:
That's right; I've seen the classes you've noted. TBH, I didn't have the header file in my first impl; I decided to separate the code into multiple files to allow for easier growth over time if needed; moreover, I wanted to see what reviewers/committers would think & update the PR accordingly.